File: AndroidDev/app/src/main/java/com/tutorials/hackro/androiddev/data/model/reddit/MediaEmbedLayerData.java

package com.tutorials.hackro.androiddev.data.model.reddit;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MediaEmbedLayerData {

    @SerializedName("content")
    @Expose
    private String content;
    @SerializedName("width")
    @Expose
    private int width;
    @SerializedName("scrolling")
    @Expose
    private boolean scrolling;
    @SerializedName("height")
    @Expose
    private int height;

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }

    public boolean isScrolling() {
        return scrolling;
    }

    public void setScrolling(boolean scrolling) {
        this.scrolling = scrolling;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

}

File: AndroidDev/app/src/main/java/com/tutorials/hackro/androiddev/data/model/reddit/MediaEmbedLayerData.kt
package com.tutorials.hackro.androiddev.data.model.reddit

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MediaEmbedLayerData {

    @SerializedName("content")
    @Expose
    var content: String? = null
    @SerializedName("width")
    @Expose
    var width: Int = 0
    @SerializedName("scrolling")
    @Expose
    var isScrolling: Boolean = false
    @SerializedName("height")
    @Expose
    var height: Int = 0

}
